variables.tf

```diff
@@ -48,4 +45,36 @@ variable "tags" {
   type = map(string)
   description = "A map of AWS Tags to attach to each resource created"
   default = {}
+}
+variable "container_url" {
+  type = string
+}
+variable "task_cpu" {
+  type        = number
+  description = "CPU for the task definition. Defaults to 256."
+  default     = 256
+}
+variable "task_memory" {
+  type        = number
+  description = "Memory for the task definition. Defaults to 512."
+  default     = 512
+}
+variable "security_groups" {
+  type = list(string)
+  description = "Extra security groups"
+  default = []
+}
+variable "vpn_to_campus" {
+  type = bool
+  description = "VPN to Campus"
+  default = false
+}
+variable "log_retention_in_days" {
+  type = number
+  default = 7
+}
+variable "health_check_grace_period" {
+  type        = number
+  description = "Health check grace period in seconds. Defaults to 0."
+  default     = 0
```

Comment:
If we're copying defaults from lower modules...I think you can set the default to `null` and Terraform will act like you didn't set the line at all, and the lower module's default will be used.
I don't know if this is better or not, but it's an option.

Reply:
If I set the default as null it tries to set the value as null